ui/core/src/components/document/Verification.vue

```diff
@@ -15,11 +15,16 @@
             </span>
         </p>
       </div>
-      <div class="stats col-sm-12 p-0" v-show="hasDocuments">
+      <div class="stats col-sm-12 p-0" v-show="hasDocuments && !hasPreinitError">
         <verification-file-entry :singleFile="files.length === 1" v-for="file in files" :key="file.name" :file="file"
                                  :wallet="wallet"
                                  :thumbnail="thumbnail"></verification-file-entry>
       </div>
+      <div class="stats col-sm-12 p-0" v-show="hasPreinitError">
```

Comment:
Why are we using the same `verification-file-entry` component twice in a different way? Does it not always expect a `file` object? Perhaps we should have a separate component for showing network errors.